ddtrace/contrib/anthropic/patch.py

```diff
@@ -101,8 +101,13 @@ def traced_chat_model_generate(anthropic, pin, func, instance, args, kwargs):
             handle_non_streamed_response(integration, chat_completions, args, kwargs, span)
     except Exception:
         span.set_exc_info(*sys.exc_info())
+        if integration.is_pc_sampled_llmobs(span):
+            integration.llmobs_set_tags(span=span, resp=chat_completions, args=args, kwargs=kwargs)
```

Comment:
This isn't necessary, the `finally` block should always execute

Reply:
removed